src/Controller/EnMarche/ReferentController.php

```diff
@@ -645,10 +647,52 @@ public function municipalVolunteerAction(
         MunicipalExporter $municipalExporter,
         UserInterface $referent
     ): Response {
-        return $this->render('referent/municipal/volunteer_request.html.twig', [
+        return $this->render('referent/municipal/volunteer/list.html.twig', [
             'volunteerListJson' => $municipalExporter->exportVolunteerAsJson(
                 $volunteerRequestRepository->findForReferent($referent)
             ),
         ]);
     }
+
+    /**
+     * @Route(
+     *     path="/municipale/candidature-colistiers/{uuid}/detail",
+     *     name="app_referent_municipal_running_mate_request_detail",
+     *     requirements={
+     *         "uuid": "%pattern_uuid%",
+     *     },
+     *     methods={"GET"},
+     * )
+     *
+     * @Security("is_granted('ROLE_REFERENT')")
+     *
+     * @Entity("runningMateRequest", expr="repository.findOneByUuid(uuid)")
```

Comment:
is it not working without? 🤔

Reply:
well, it seems to work. I thought this was mandatory to get it working.